src/ApiClient.php

```diff
@@ -33,15 +29,10 @@
  * It handles requesting and setting up what is necessary to execute calls to the endpoints.
  *
  * @see ApiClientInterface
- * @see ConfigurationAwareTrait
  */
 class ApiClient implements ApiClientInterface
 {
-    use ConfigurationAwareTrait;
```

Comment:
I moved all the configuration/token stuff to the factory so I wouldn't have to pass it back and forth.